### ISSUE TITLE
axsdist units are unclear

### ISSUE THREAD

**Issue by ctslater** [dirac-institute/AXS#17]
Ernesto Castillo asks a very good question: what are the units on axsdist? This seems to be computed by `calcGnom` in `FrameFunctions.scala`, but I can't follow what the final formula is producing.

**Comment by Dav-v** [comment]
Hi, are there any updates on this? Looking at the functions `calcGnom()` and `degToGnom()` and the conversion `gnom_dist = degToGnom(r)` ( gnom_dist = tan(rad(r)) ), could this be a gnomonic projection? Is there something in the documentation or in other external sources that explains the method and the formulas used to determine the distance?

**Comment by zecevicp** [comment]
If we take the [Gnomonic projection equations](https://mathworld.wolfram.com/GnomonicProjection.html) and set phi_1 and lambda_0 both to zero, (and also substitute lambda for abs(ra_2-ra_1) and phi for abs(dec_2-dec_1)), that would be as though the projection is centered on ra_1 and dec_1. That's what was implemented. I hope that makes sense for you?

**Comment by Dav-v** [comment]
Yes, thank you very much @zecevicp . I was also wondering, is there a particular reason why the UDFs are written in Scala? Wouldn't be possible to implement calcGnom() directly in Python as a PandasUDF or using the built-in SQL functions? Thanks again
